core/relayMsgs.go

```diff
@@ -161,6 +172,14 @@ func (r *RelayMsgs) Send(src, dst Chain) {
 	r.DstMsgIDs = dstMsgIDs
 }
 
+func msgsToLoggable(msgs []sdk.Msg) []string {
+	var ret []string
+	for _, msg := range msgs {
+		ret = append(ret, fmt.Sprintf("%T", msg))
```

Comment:
@siburu Do we need only type info of msg?

Reply:
@bluele Thank you.
When I wrote this code, I didn't know how to log `sdk.Msg` without causing panics.

Now I know two different ways to do this.
- use jsonpb.Marshaler
- use fmt.Sprint

The former one generates more readable outputs, so I will fix this code by using jsonpb.Marshaler.

Reply:
Ah ... a plain `jsonpb.Marshaler{}` cannot marshal `sdk.Msg` that contains Any to JSON.
A ProtoCodec is required to do this, but `RelayMsgs` doesn't have an access to ProtoCodec.

So I will use `fmt.Sprint` to fix this code.